x-pack/plugins/rollup/public/index_pattern_creation/rollup_index_pattern_creation_config.js

```diff
@@ -47,69 +45,50 @@ export class RollupIndexPatternCreationConfig extends IndexPatternCreationConfig
     } : null;
   }
 
-  getIndexPatternCreationQuery = () => {
-    return {
-      query: {
-        exists: {
-          field: "_rollup.version"
-        }
-      },
-    };
-  };
-
-  illegalCharacters = (characters = []) => {
-    return ['*'].concat(characters);
+  isRollupIndex = (indexName) => {
```

Comment:
This code is pretty simple and straightforward, but because it's so far removed from the context in which it's consumed, maybe some unit tests would be useful for establishing intention and expected use cases? I'm mostly thinking for the `getIndexPatternCreationOption`, `isRollupIndex`, `getIndexTags`, and `checkIndicesForErrors` methods. WDYT?

Reply:
👍 will add tests next pass too